metrics.md

```diff
@@ -0,0 +1,47 @@
+ 
+#### Gluster prometheus end point schema: 
+
+#### Volume Metrics : 
+
+|Gluster Promethus schema| example| codes |
+|--------|-----------|---|
+|gluster_peer_status |	gluster_peer_status{nodeid="xxxxx"} 1 |	1 - connected, 0 - disconnected	
+|gluster_volume_status|	gluster_volume_status{volume="xxx"} 1	| 1 - started, 0 - stopped, 2 - starting, 3 - stopping	
+|gluster_volume_fop_max_latency	| gluster_volume_max_latency{volume=""xxx"", brick_path=""xxxx"",fop=""xxx""}  800970.00| 	
+|gluster_volume_fop_min_latency	|gluster_volume_fop_min_latency{volume="xxx", brick_path="xxx", fop="xxx"} 21.00	|
+|gluster_volume_fop_avg_latency|	gluster_volume_fop_avg_latency{volume="xxx", brick_path="xxx", fop="xxx"} 1132.00|	
+|gluster_volume_fop_calls |gluster_volume_fop_calls{volume="xxx". brick_path="xxx", fop="xxx"} 13	|
+|gluster_volume_fop_percentage_latency| gluster_volume_fop_percentage_latency(volume="xxx", brick_path="xxx", fop="xxx"} 13	|
+|gluster_volume_total_bytes| gluster_volume_total_bytes{volume="xxx"}  104857600	|
+|gluster_volume_used_bytes| gluster_volume_used_bytes{volume="xxx"}  4857600	|
+|gluster_volume_available_bytes|	gluster_volume_available_bytes{volume="xxx"} 9856780	|
+
+#### Brick Metrics:
```

Comment:
@vredara Available brick level metric names as below

gluster_brick_capacity_used
gluster_brick_capacity_free
gluster_brick_capacity_total
gluster_brick_inodes_total
gluster_brick_inodes_free
gluster_brick_inodes_used
gluster_subvol_capacity_used

The names should be mentioned accordingly here

Reply:
ack. What is gluster_subvol_capacity_used?

Reply:
This is at subvolume level used capacity. For each subvolume it would be reported and based on that overall volume level utilization could be derived.